### PR TITLE
feat: add bitcoincash

### PR DESCRIPTION
**Pull request by kaladinlight**

### CODE REVIEW

File: packages/chain-adapters/src/utils/utxoUtils.ts [hunk 1]
```diff
@@ -38,7 +45,7 @@ export const utxoAccountParams = (
         scriptType: BTCInputScriptType.SpendAddress,
         bip44Params: {
           purpose: 44,
-          coinType: 3,
+          coinType: Number(ASSET_REFERENCE.Dogecoin),
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/utils/utxoUtils.ts:48]: 👍

File: packages/types/src/base.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export enum KnownChainIds {
   EthereumMainnet = 'eip155:1',
   AvalancheMainnet = 'eip155:43114',
   BitcoinMainnet = 'bip122:000000000019d6689c085ae165831e93',
+  BitcoincashMainnet = 'bip122:000000000000000000651ef99cb9fcbe',
```
Comment (gomesalexandre) [inline packages/types/src/base.ts:13]: To keep the casing consistent? e.g `bitcoincash` when lowercased, `bitcoinCash` when camelcased... `BitcoinCash` when PascalCased? 

```suggestion
  BitcoinCashMainnet = 'bip122:000000000000000000651ef99cb9fcbe',
```

Comment (kaladinlight) [reply packages/types/src/base.ts]: updated

**Review by 0xdef1cafe — APPROVED** [review]

**Review by elmutt — APPROVED** [review]

**Review by gomesalexandre — APPROVED** [review]
Untested but gave it a few review passes and logic looks sound - anything weirdy here would be spotted in web but this effectively looks the same as other UTXO implementations

**Pull request merged**